.storybook/config.js

```diff
@@ -43,6 +43,13 @@ const GlobalStyle = createGlobalStyle`
     font-weight: 600;
     src: url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff2') format('woff2'), url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff') format('woff');
   }
+  @font-face {
```

Comment:
I added this so the Reith Sans Bold font will load in Storybook. Though issue https://github.com/bbc/psammead/issues/285 updating font-faces within Psammead utilities will be used to update the font-faces in the Storybook config.

Reply:
Do you need to update the screenshot in this PR description?